dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -339,20 +339,22 @@ private DependencyGraphResult levelOrder(
             for (ArtifactResult artifactResult :
                 resolutionException.getResult().getArtifactResults()) {
               if (artifactResult.getArtifact() == null) {
-                DependencyNode failedDependencyNode = artifactResult.getRequest().getDependencyNode();
-                ExceptionAndPath failure =
-                    ExceptionAndPath.create(parentNodes, failedDependencyNode, resolutionException);
-                artifactProblems.add(new UnresolvableArtifactProblem(failure.getPath()));
+                DependencyNode failedDependencyNode =
+                    artifactResult.getRequest().getDependencyNode();
+                
+                List<DependencyNode> fullPath = makeFullPath(parentNodes, failedDependencyNode);
+                
+                artifactProblems.add(
+                    new UnresolvableArtifactProblem(
```

Comment:
```suggestion
                    new UnresolvableArtifactProblem(fullPath)
```

Then the new constructor is not needed.

Reply:
removed